Sources/MongoKitten/MongoDatabase.swift

```diff
@@ -76,9 +80,10 @@ public class MongoDatabase {
     ///
     /// - parameter settings: The connection settings, which must include a database name
     public static func lazyConnect(
-        to settings: ConnectionSettings
+        to settings: ConnectionSettings,
+        logger: Logger? = nil
```

Comment:
Can we make this non-optional with old logger being the default argument, rather than a nullable argument?